doc/old/working_with_overlays.rst

```diff
@@ -61,6 +61,13 @@ overlay elements you're interested in::
   >>> arr.shape
   (484, 484)
 
+One thing to remember when dealing with *Overlay Data* is that the top left
+of the overlay doesn't necessarily have to line up with the top left of the
+related *Pixel Data*. The actual offset between them can be determined from
+(60xx,0050) *Overlay Origin*, where a value of ``[1, 1]`` indicates that
+the top left pixels are aligned and a value of ``[0, 0]`` indicates that the
+overlay pixels start 1 row above and 1 row to the left of the image pixels.
+
```

Comment:
Just a suggestion -- this has the feel of a Warning block, and would be more visible that way.

Reply:
I think I'll leave it as is